lambda/import/embeddings.js

```diff
@@ -0,0 +1,55 @@
+const aws = require('aws-sdk');
```

Comment:
@rstrahan this file is identical to the one in es-proxy-layer. Can we add the esproxy layer to the import lambdas and call from there?

Reply:
done